doc/CustomEventConnector..md

```diff
@@ -0,0 +1,77 @@
+# reshuffle
+
+[Code](https://github.com/reshufflehq/reshuffle) |
+[npm](https://www.npmjs.com/package/reshuffle) |
+[Code sample](https://github.com/reshufflehq/reshuffle/examples/customEvent)
+
+
+### Reshuffle Custom Event Connector
+
+This package contains a [Reshuffle](https://github.com/reshufflehq/reshuffle)
+connector that can fire a customized event. Developer can fire it from a script.
```

Comment:
I think `custom` instead of `customized` is ok?
Also the last sentence can be better